### PR TITLE
Increase partition size to 30

### PR DESCRIPTION
**Pull request by DrewScoggins**
The mono interpreter tests take a very long time to finish. In order
to mitigate this we are going to split the workitems up further and
have each machine do less work. Since we have a large pool of empty
machines this should be fine.

### CODE REVIEW

**Review by billwert — APPROVED** [review]

**Comment by akoeplinger** [comment]
Hey, we had to disable the long interpreter runs with https://github.com/dotnet/runtime/pull/38030, you can turn them back on in this PR if you think this would solve the issue :)

**Comment by SamMonoRT** [comment]
@DrewScoggins - this PR disabled the interpreter runs : https://github.com/dotnet/runtime/pull/38030   You might want to add it back after proper approvals.

**Comment by danmosemsft** [comment]
@DrewScoggins are you still working on this PR or should it be closed?

**Comment by DrewScoggins** [comment]
Sorry, this must have gotten dealt with in another PR, because we have had this for a long time. I will close this as it is out of date.

**Pull request closed without merging**
